src/main/java/com/shippingcostestimator/enterprise/service/ShipmentServiceStub.java

```diff
@@ -31,7 +31,7 @@ public ShipmentServiceStub(IShipmentDAO shipmentDAO){
     /***
      * Returns a Shipment object with a specified integer ID.
      * @param id
-     * @return
+     * @return A shipment estimate based on specified ShipmentId.
      */
```

Comment:
Added return description JavaDoc [Ref.](https://medium.com/@rhamedy/a-short-summary-of-java-coding-best-practices-31283d0167d3)